ts/twitter/src/lib/components/more-popup/more-popup.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+import { Flex, IconButton, Menu, Portal, Text } from "@chakra-ui/react";
+import { useBreakpointValue } from "@chakra-ui/react";
+import { CiCircleMore } from "react-icons/ci";
+import { DiAptana } from "react-icons/di";
+import { FaMoneyBills, FaPodcast } from "react-icons/fa6";
+import { MdOpenInNew } from "react-icons/md";
+
+import { useColorModeValue } from "../ui/color-mode";
+import { Tooltip } from "../ui/tooltip";
+
+import { Box } from "@chakra-ui/react";
+
+interface MenuItemProps {
+  label: string;
+  icon: React.ReactElement;
+}
+
+export const MorePopup: React.FC = () => {
+  const isTooltipDisabled = useBreakpointValue({ base: false, xl: true });
+  const menuItems: MenuItemProps[] = [
+    {
+      label: "Monetization",
+      icon: <FaMoneyBills />,
+    },
+    {
+      label: "Ads",
+      icon: <MdOpenInNew />,
+    },
+    {
+      label: "Create your Space",
+      icon: <FaPodcast />,
+    },
+    {
+      label: "Settings and privacy",
+      icon: <DiAptana />,
+    },
+  ];
+
+  return (
+    <Box position="relative">
+      <Menu.Root>
+        <Menu.Trigger asChild>
+          <Flex alignItems="center" p="12px" width="auto">
+            <Tooltip content={"More"} disabled={isTooltipDisabled}>
```

Comment:
Curly braces should only be used when the value is dynamic or a JavaScript expression.
```suggestion
            <Tooltip content="More" disabled={isTooltipDisabled}>
```